cmd/workflow-controller/main.go

```diff
@@ -103,7 +103,7 @@ func NewRootCommand() *cobra.Command {
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
 	command.Flags().IntVar(&glogLevel, "gloglevel", 0, "Set the glog logging level")
 	command.Flags().IntVar(&workflowWorkers, "workflow-workers", 8, "Number of workflow workers")
```

Comment:
workflow-workers should be increased as well. That would have biggest benefit

Reply:
do you have suggestion for new values?